Source/UnitTests/TestUtil/BoogieTestBase.cs

```diff
@@ -10,7 +10,9 @@ public BoogieTestBase()
 
       // FIXME: THIS IS A HACK. Boogie's methods
       // depend on its command line parser being set!
-      CommandLineOptions.Install(new Microsoft.Boogie.CommandLineOptions());
+
+      // TODO FIX
+      //CommandLineOptions.Install(new Microsoft.Boogie.CommandLineOptions());
```

Comment:
Is there work still needed here? I imagine that, since this is in the tests, they'd be breaking if this was still necessary.

Reply:
Good point. Will remove this class.